vsphere/datadog_checks/vsphere/event.py

```diff
@@ -33,13 +33,11 @@
     'VmSuspendedEvent': [],
 }
 
-ALLOWED_EVENTS = [getattr(vim.event, event_type) for event_type in EXCLUDE_FILTERS.keys()]
-
 
 class VSphereEvent(object):
     UNKNOWN = 'unknown'
 
-    def __init__(self, raw_event, event_config, tags, event_resource_filters):
+    def __init__(self, raw_event, event_config, tags, event_resource_filters, exclude_filters=EXCLUDE_FILTERS):
```

Comment:
```suggestion
    def __init__(self, raw_event, event_config, tags, event_resource_filters, exclude_filters):
```
I don't think we need a default here since we have it in the config

Reply:
Removed the default.

Reply:
Ah I forgot we use this for the legacy check too! This is definitely for a future PR, but I think we should eventually just fully split out the legacy code